tests/vm_test.go

```diff
@@ -2109,3 +2110,37 @@ func stopVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.Virtua
 
 	return vm
 }
+
+func newRandomVirtualMachineInstanceWithFileDisk(namespace string, accessMode k8sv1.PersistentVolumeAccessMode) (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
+	sc, foundSC := libstorage.GetRWOFileSystemStorageClass()
+	if accessMode == k8sv1.ReadWriteMany {
+		sc, foundSC = libstorage.GetRWXFileSystemStorageClass()
+	}
```

Comment:
This is redundant.

Reply:
but it's different, line 2115 uses `GetRWO` and line 2117 uses `GetRWX`

Reply:
@xpivarc Why is this redundant? can you please drop a few words? is this because we will fail anyway in case of improper setup or improper treatment of the test decorators?

Reply:
It is redundant because the function is used only once and the only input is RWO, therefore it doesn't need to expose the argument at all and can be further simplified.